src/UserNotes/UserNote.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace phpweb\UserNotes;
+
+class UserNote
+{
+    /** @var string $id */
+    public $id;
+    /** @var string $sect */
+    public $sect;
+    /** @var string $rate */
+    public $rate;
+    /** @var string $ts */
+    public $ts;
+    /** @var string $user */
+    public $user;
+    /** @var string $text */
+    public $text;
+    /** @var int $upvotes */
+    public $upvotes;
+    /** @var int $downvotes */
+    public $downvotes;
```

Comment:
How do you feel about marking these as `private` and adding accessors instead?

```suggestion
-    /** @var string $id */
-    public $id;
-    /** @var string $sect */
-    public $sect;
-    /** @var string $rate */
-    public $rate;
-    /** @var string $ts */
-    public $ts;
-    /** @var string $user */
-    public $user;
-    /** @var string $text */
-    public $text;
-    /** @var int $upvotes */
-    public $upvotes;
-    /** @var int $downvotes */
-    public $downvotes;
+    private $id;
+    private $sect;
+    private $rate;
+    private $ts;
+    private $user;
+    private $text;
+    private $upvotes;
+    private $downvotes;
```